spec/controllers/announcements_controller_spec.rb

```diff
@@ -15,6 +15,21 @@
     @app_config = FactoryGirl.create(:app_config)
   end
 
+  shared_examples 'can hide announcement' do
```

Comment:
This shared example block should likely be under the relevant `describe` block as it's not applicable to most of the specs in the file.